nipst/nipst.go

```diff
@@ -228,9 +235,13 @@ func (nb *NIPSTBuilder) IsPostInitialized() bool {
 	return true
 }
 
-func (nb *NIPSTBuilder) InitializePost() (*types.PostProof, error) {
+func (nb *NIPSTBuilder) InitializePost(logicalDrive string, commitmentSize uint64) (*types.PostProof, error) {
 	defTimeout := 5 * time.Second // TODO: replace temporary solution
 
+	nb.postCfg.DataDir = logicalDrive
+	nb.postCfg.SpacePerUnit = commitmentSize
+	nb.postProver.SetPostParams(logicalDrive, commitmentSize)
```

Comment:
Having post cfg on both the nipst builder and the post prover looks confusing. See if you can have in one place only.